populate_db.py

```diff
@@ -0,0 +1,107 @@
+import pyphen, requests, json
+#pyphen.language_fallback('ipa_en')
+ipa_dic = pyphen.Pyphen(lang='ipa_en')
+spelling_dic = pyphen.Pyphen(lang='en')
+
+URL = "http://127.0.0.1:5000"
+LANGUAGE = "english"
+
+
+print(ipa_dic.inserted("sʌbdʒɛktɪv"))  # use dark magic space trick in the future
+
+wordlist = "allprons.txt"
+data = open(wordlist, mode="r", encoding="utf8").read()
+
+separators = data.splitlines()[0].split()
+
+raw_data = data.splitlines()[1:]
+
+#print(raw_data)
+
+stop_chars ="̈ᵊ̰̚˞͡ˀ̪̃ʔʰ˩˥ʲ̟̯̬​ˠ̞ʷʱ̝ˁ̻cǀˑ̥̹ˢˈˈ̺͡"
+
+vowels = "iyɨʉɯuɪʏɪʊeøɘɵɤoe̞ø̞əɤ̞o̞ɛœɜɞʌɔæɐaɶäɑɒ"
+
+#replaces rare, unusual sounds with close equivalents in // notation
+simplifying = {'ä': 'a', 'ɟ': 'g', 'á': 'a', "ɚ": 'ər', "ʍ": "w", "ɝ": "ər", "ɘ": "ɪ", "ɹ": "r", "õ": "ɔn", "ɱ": "m",
+               "œ": "", "ʋ": "w", "ɯ": "ə", "ɻ": "r", "ɧ": "ʃ", "ɬ": "l", "ã": "a", "ɕ": "ʃ", "ɲ": "nj", "ç": "h",
+               "ẽ": "e", "ɦ": "h", "g": "ɡ", "ĭ": "ɪ", "ĩ": "ɪ", "ʏ": "ʊ", "χ": "x", "ŏ": "ə", "ɥ": "ju", "ɵ": "ɔ",
+               "ʎ": "j", "ʝ": "j", "έ": "e"}
+
+def simplify(string):
+    if string in simplifying:
```

Comment:
a one liner: `return simplifying.get(string, string)`

also here you are mixing with global variable... not sure it's a good idea.